first_script.py

```diff
@@ -522,4 +522,81 @@ def make_spider (row, title, color):
 plt.savefig(mypath +'corticothalamic_FSL.png', dpi=None, facecolor='w', edgecolor='w',
         orientation='landscape', papertype=None, format=None,
         transparent=False, bbox_inches=None, pad_inches=0.1,
-        frameon=None, metadata=None)
\ No newline at end of file
+        frameon=None, metadata=None)
+
+##############################################################################
+# predict presence of verbal perseveration by applying random forrest analysis
+
+#verbal perseverations early postop
+
+X=ANTS_ratio.iloc[:, 0:46]
+Y=behavior.iloc[:,3]
+
+#remove subject with nan value from both datasets (here the second line)
+X=X.drop(X.index[1])
+Y=Y.drop(Y.index[1])
+
+from sklearn.model_selection import train_test_split
+X_train, X_test, Y_train, Y_test = train_test_split (X, Y, test_size=0.2, random_state=0)
+
+#this normalisation step makes the performance drop
+#from sklearn.preprocessing import StandardScaler
+#sc = StandardScaler ()
+#X_train= sc.fit_transform (X_train)
+#X_test = sc.fit_transform (X_test)
+
```

Comment:
Indeed you should never normalize train and test independently. You can remove these lines.

Reply:
Ok sorry I thought I was applying the same transformation to both, but obviously this was not the case...